src/deviation-check/deviation-check.ts

```diff
@@ -60,13 +62,13 @@ export const checkUpdateCondition = (
 ): boolean => {
   if (onChainTimestamp === 0n && offChainTimestamp > 0) return true;
   if (
-    deviationThreshold &&
```

Comment:
I believe I wanted to prevent cases when the `deviationThreshold` and `heartbeatInterval` were uninitialised, but not sure if this is even possible.

But I don't see an use case for either of the values to be set to 0.

Reply:
Maybe there will be a feed that expected to be updated in each cycle by setting heartbeat interval or deviation threshold to 0.

So I'm keeping my changes if the reasoning is incase of uninitialization.

Reply:
If that is legit requirement, then those lines should imo be removed.